src/SWP/TemplateEngineBundle/Form/Type/ContainerType.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Template Engine Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\TemplateEngineBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+
+class ContainerType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add('name', null, array(
+            'required' => false,
+        ));
+        $builder->add('height', null, array(
+            'required' => false,
+        ));
+        $builder->add('width', null, array(
+            'required' => false,
+        ));
+        $builder->add('styles', null, array(
```

Comment:
`$builder` object has fluent interface, you dont need to do `$builder->add` all() the time